misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestAnnotationProcessor.java

```diff
@@ -137,7 +173,45 @@ public void process(FunctionNode functionNode, List<AnnotationAttachmentNode> an
                             vals[1] = value;
                         }
                     });
-                    suite.addMockFunction(vals[0] + MOCK_ANNOTATION_DELIMITER + vals[1], functionName);
+
+                    if (bLangPackage == null) {
+                        setBlangPackage(attachmentNode);
+                    }
+
+                    if (vals[0].isEmpty()) {
+                        diagnosticLog.logDiagnostic(Diagnostic.Kind.ERROR, ((BLangFunction) functionNode).pos,
+                                "Module name cannot be empty");
```

Comment:
For compile errors the standard we follow is to use lower case letters at the start of an error message.

Reply:
Also, this error will be given to the beginning of the function. Check and see if you can give the error to the correct position of the relevant field in the annotation.

Reply:
I have changed this so that it highlights the error in the annotation